source/docs/application-notes/update-frequency-impact.rst

```diff
@@ -78,3 +80,43 @@ Final States
 As can be seen, going from the RIO bus to the CANivore bus, or from 50 Hz to 250 Hz improves the accuracy of the odometry, and by a noticeable amount. Based on this, utilizing faster update frequencies and time synchronization from the CANivore should result in more accurate odometry, even for the "short" movements as shown in the gif.
 
 Data on the pose location is available for download: :download:`OdometryData.xlsx </docs/application-notes/data/frequency-impact/OdometryData.xlsx>`.
+
+After-Test Data
+---------------
+Roughly 2 weeks after this initial data was collected and the blog post written, we went back and re-verified the data for the CANivore 250 Hz and RIO 250 Hz cases to further test the impact of time synchronization.
+These tests were ran in autonomous a total of 20 times (10 for CANivore, 10 for RIO), measuring the error of the odometry against the Limelight data.
+The results are below:
```

Comment:
If this is intended to be two separate paragraphs, add another newline between them.